### ISSUE TITLE
How to add this to existing hooks?

### ISSUE THREAD

**Issue by dashmug** [awslabs/git-secrets#109]
Can I run this as a script on top of my own git hooks? I'm using https://github.com/typicode/husky and it manages my git hooks for me. Basically I can configure husky to run any terminal command as a git hook.

**Comment by andreasstieger** [comment]
Yes, you should be able to deduce the required configuration for husky from:
https://github.com/awslabs/git-secrets/blob/1.3.0/git-secrets#L195-L215

e.g. `"pre-commit": "git secrets --prepare_commit_msg_hook [...]"`
